runestone/activecode/js/activecode.js

```diff
@@ -611,7 +611,7 @@ ActiveCode.prototype.showCodelens = function () {
 
     var cl = this.codelens.firstChild;
     if (cl) {
-        div.removeChild(cl)
+        this.codelens.removeChild(cl)
```

Comment:
This is just fixing a bug. It wasn't removing it, so it wasn't doing a new call to create it with the new code.